scripts/scan-all.sh

```diff
@@ -2,7 +2,7 @@
 set -eo pipefail
 
 # Hard-coding four external images we own. Versions taken from insights-agent. Need to find a better solution here.
-images=(quay.io/fairwinds/polaris:7.4 quay.io/fairwinds/nova:v3.6 us-docker.pkg.dev/fairwinds-ops/oss/pluto:v5.16 us-docker.pkg.dev/fairwinds-ops/oss/goldilocks:v4.7)
+images=(quay.io/fairwinds/polaris:7.4 quay.io/fairwinds/nova:v3.6 us-docker.pkg.dev/fairwinds-ops/oss/pluto:v5.16 us-docker.pkg.dev/fairwinds-ops/oss/goldilocks:v4.8)
```

Comment:
is `goldilocks` upgrade supposed to be in this PR?

Reply:
Good question, I think this is just for the security scan. I maybe assumed wrong that we wanted these to be pointed at the latest versions? Should that be addressed somewhere else. This also applies to your comment above

Reply:
Yeah, I think it does make sense...

In [fairwinds-insights.yaml](https://github.com/FairwindsOps/insights-plugins/pull/786/files#diff-cd7b940b608e002b123a20a164d7dcd21dfcbe56242d68cc6d378afa5bf944be) we explicitly point out the images that we use, so they can be scanned